spec/webhookdb/replicator/brevo_email_activity_unaggregated_events_v1_spec.rb

```diff
@@ -0,0 +1,290 @@
+# frozen_string_literal: true
+
+require "support/shared_examples_for_replicators"
+
+RSpec.describe Webhookdb::Replicator::BrevoEmailActivityUnaggregatedEventsV1, :db do
```

Comment:
There are a few other behaviors I'll add support for.

Reply:
Is there another replicator that I could use as a guide / example?

Reply:
I have added test ["...prevents overwriting new data with old"](https://github.com/webhookdb/webhookdb/pull/859/commits/1d6be4fe9262f847ee4d80a9879b00bbe3325cdd)